<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TFT Helper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="resultTable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>220</width>
      <height>281</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>num</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>prob</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="outputTypeCbx">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>#target</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>#rolling</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Output r.v.</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="inputGbx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Input Parameters</string>
    </property>
    <widget class="QLabel" name="lvlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Lv:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lvlineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tierlineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="tierlabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tier:</string>
     </property>
    </widget>
    <widget class="QLabel" name="fixRVlabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>FixRV:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fixRVEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="targetDrawnNumEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="targetDrawnNumlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>#target drawn:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sameTierNumEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="sameTierNumlabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>#same Tier:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="runBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="meanLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mean:</string>
    </property>
   </widget>
   <widget class="QLabel" name="stdLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>std:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionChinese"/>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
